ConsoleApp1/Snake.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Timers;
+
+namespace ConsoleApp1
+{
+    public class Snake
+    {
+        public static int score { get; private set; } = 0;
+        public char SnakeChar { get; set; }
+        public static Point point { get; private set; }
+        public static List<Point> Body { get; private set; }
+
+        public Snake()
+        {
+            point = new Point(Board.Size / 2, Board.Size / 2);
+            Body = new List<Point>
+            {
+                new Point(Board.Size / 2, Board.Size / 2),
+                new Point((Board.Size / 2), Board.Size / 2),
+                new Point((Board.Size / 2), Board.Size / 2),
+            };
+        }
+
+        public void MoveSnake(object? sender, ElapsedEventArgs e)
+        {
+            Direction.GetDirection();
+            int side = (int)Direction.eDirection;            
+            switch (side)
+            {
+                case 0: Body.Add(new Point(Body.Last().x, Body.Last().y - 1)); SnakeChar = '^'; break;
+                case 1: Body.Add(new Point(Body.Last().x, Body.Last().y + 1)); SnakeChar = 'V'; break;
+                case 2: Body.Add(new Point(Body.Last().x - 1, Body.Last().y)); SnakeChar = '<'; break;
+                case 3: Body.Add(new Point(Body.Last().x + 1, Body.Last().y)); SnakeChar = '>'; break;
+            }
+
+            Console.SetCursorPosition(Body[0].x, Body[0].y);
+            Console.Write(" ");
+            Body.RemoveAt(0);
+            CheckBoard();            
+            Render();
+            CheckFruit();
+            CheckIfCol();
+
+        }
+
+        public void EatFruit(Fruit fruit)
+        {
+            Direction.EDirection dir = Direction.eDirection;
+            switch (dir)
+            {
+                case Direction.EDirection.North:
+                    Body.Add(new Point(Body.Last().x, Body.Last().y - 1));
+                    break;
+                case Direction.EDirection.South:
+                    Body.Add(new Point(Body.Last().x, Body.Last().y + 1));
+                    break;
+                case Direction.EDirection.West:
+                    Body.Add(new Point(Body.Last().x - 1, Body.Last().y));
+                    break;
+                case Direction.EDirection.East:
+                    Body.Add(new Point(Body.Last().x + 1, Body.Last().y));
+                    break;
+            }
+            score++;
```

Comment:
you can calculate a score based on snake length - 3, so additional score variable is extra